src/NuGet.Indexing/StorageLoader.cs

```diff
@@ -42,5 +52,23 @@ public JsonReader GetReader(string name)
                 throw;
             }
         }
+
+        public async Task<bool> Reload()
```

Comment:
Cool. Looks like we can leverage this to dynamically update the container name from another source.